REQUIRE

```diff
@@ -7,3 +7,5 @@ SortingAlgorithms
 Reexport
 Compat 0.19.0
 FileIO 0.1.2
+CSV
+DataStreams
```

Comment:
Is DataStreams necessary here? Seems you're only using CSV, unless I'm missing something.